packages/spright-components/README.md

```diff
@@ -0,0 +1,9 @@
+<div align="center">
+    <p><b>ni | spright</b></p>
+</div>
+
+# Spright Components
```

Comment:
I'm debating whether we want to provide an equivalent of the Getting Started section in the nimble-components README. Would it be useful to say something like "Follow the instructions in the nimble-components README, replacing "nimble" with "spright" as appropriate"?

I'm guessing that will be obvious to most users of the package, but it couldn't hurt in case people stumble on Spright before Nimble.

Reply:
I think each public package should have a very clear README and CONTRIBUTING each with a Getting Started started section that is step-by-step and completable. I can own adding that after this PR goes in because I think nimble's and spright's need a clean-up for the latest era.